packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1067,8 +1071,9 @@ class ShapeBorderClipper extends CustomClipper<Path> {
   /// the border will not need the text direction to paint itself.
   const ShapeBorderClipper({
     @required this.shape,
+    Clip clip = defaultClipBehavior, // ignore: deprecated_member_use
     this.textDirection,
-  }) : assert(shape != null);
+  }) : assert(shape != null), assert(clip != null), super(clip: clip);
```

Comment:
no need to check clip if the superclass is going to check it

Reply:
Done.